reducers/views/rhs.js

```diff
@@ -115,6 +115,8 @@ function searchTerms(state = '', action) {
 function pluggableId(state = '', action) {
     switch (action.type) {
     case ActionTypes.UPDATE_RHS_STATE:
+    case ActionTypes.SELECT_POST:
```

Comment:
Should these be a separate case in the switch statement? I don't think `SELECT_POST` or `SELECT_POST_CARD` pass a `state` field at all.

Reply:
It was mainly to reset the state back to `''` so `action.state` wasn't being used in those cases. But I think it's cleaner to give it its own case.

Reply:
Yeah, I like that better. The previous way worked mostly because action happens to look different in those two cases, but I do agree that it's cleaner to separate them entirely.